Paralayout/PixelRounding.swift

```diff
@@ -151,3 +151,33 @@ extension CGSize {
     }
 
 }
+
+extension CGRect {
+
+    /// Outsets the rect, if necessary, to snap to the nearest pixel at the specified scale.
+    /// - parameter scaleFactor: The pixel scale to use, e.g. a UIScreen, UIView, or explicit value (pass `0` to *not*
+    /// snap to pixel).
+    /// - returns: A new rect with pixel-aligned boundaries, enclosing the original rect.
+    public func expandToPixel(_ scaleFactor: ScaleFactorProviding) -> CGRect {
```

Comment:
Should be `expandedToPixel` (like `lowercased`) since it returns a new copy and doesn't modify-in-place. (Same below.)

Reply:
Agreed. Left this alone since it was an existing part of the API, but maybe worth updating now as part of 1.0.

Reply:
Actually, I think all of these rounding methods should be renamed (`roundToPixel` -> `roundedToPixel`). Filed #42 since this is a bit out of scope for this PR.

Reply:
@efirestone Happy to move forward with this PR and follow up on the rounding methods later?